cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/patch/UserPatchDTO.java

```diff
@@ -0,0 +1,38 @@
+package com.cloudesire.platform.apiclient.dto.model.patch;
+
+import com.cloudesire.platform.apiclient.dto.model.dto.DTO;
+
+import javax.validation.constraints.NotNull;
+
+public class UserPatchDTO extends DTO
+{
+    @NotNull
+    private Action action;
+
+    public UserPatchDTO( Action action )
+    {
+        this.action = action;
+    }
+
+    public UserPatchDTO()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "action" is marked "javax.validation.constraints.NotNull" but is not initialized in this constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2637)